code/index.html

```diff
@@ -17,21 +18,20 @@
         <img id="killerImage" src="" height="100" />
         <p id="killerDescription"></p>
       </article>
+      <article class="room-deck" id="roomCard">
+        <p>ROOM</p>
+        <p id="roomName"></p>
+        <img id="roomImage" src="" height="100" />
+      </article>
       <article class="weapon-deck" id="weaponCard">
         <p>WEAPON</p>
         <p id="weaponName"></p>
         <p id="weaponWeight"></p>
```

Comment:
Maybe you could add some comments even in the HTML code to explain what the HTML does 😄